translations/stripes-data-transfer-components/en_US.json

```diff
@@ -1 +1,3 @@
-{}
+{
```

Comment:
No need for changes in this file. Localize jenkins job will do it for us. It is not recommended to update the file where localize will do the job.